config/authorization/config.lisp

```diff
@@ -84,6 +86,14 @@
   ("foaf:Group"
     -> "foaf:member"))
 
+(define-graph holidays ("http://mu.semte.ch/graphs/holidays")
```

Comment:
I would use `http://mu.semte.ch/graphs/redpencil` as URI for the graph. That's also were you insert the data in the migration. I assume in the future it will be stored in a graph per employee like the work logs.

Probably best to give the graph a more general name such that other kinds of data that can be viewed by normal users and edited by admins can also be configured in this graph. My best proposals atm are `timesheet-admin` or `timesheet-meta`.

I would move the definition up such that is near the `timesheet` graph.